lib/cretonne/src/ir/dfg.rs

```diff
@@ -657,6 +665,28 @@ impl DataFlowGraph {
                         })
     }
 
+    /// Removes the argument `val` from `ebb`. Returns true if the removal is sucessful.
+    pub fn remove_ebb_arg(&mut self, ebb: Ebb, val: Value) -> bool {
```

Comment:
I created a pull request concerning only the DFG changes here: https://github.com/stoklund/cretonne/pull/95.